piet-web/src/lib.rs

```diff
@@ -26,6 +26,29 @@ pub enum StrokeStyle {
     Default,
 }
 
+#[derive(Clone)]
+pub struct WebFont {
+    family: String,
+    weight: u32,
+    // Maybe this should be enum? Everything is stringly typed...
+    style: &'static str,
```

Comment:
Is this meant to model this? https://developer.mozilla.org/en-US/docs/Web/CSS/font-style If so, I think an enum is better. Since JavaScript doesn't have first class enums, generally what would be modeled as an enum in Rust (or even C) is modeled through strings. I think having a policy of exposing these as enums is a good idea.

Reply:
Agreed. Would it be better to do an `as_string()` method on the enum, or impl std::fmt::Display on it?

Reply:
The docs say "Display is for user-facing output" which I would not classify this as. `as_string` could work. Another option, since I believe this is only meant to be used in one place is to just have a function `font_str` which does a match on the style.

Reply:
I'll go with that then.